src/components/table-content-wrapper/table-content-wrapper.jsx

```diff
@@ -40,6 +41,8 @@ class TableContentWrapper extends Component {
       trProps,
       borderlessRows,
     } = this.props;
+    const captionHidden = !captionVisibleMobile && !captionVisibleDesktop;
+    const captionVisible = captionVisibleMobile || captionVisibleDesktop;
```

Comment:
`captionHidden` is a bit redundant here, it's the same as if `captionVisible` is false.